docker/Dockerfile

```
@@ -31,7 +30,7 @@ RUN chmod 755 $HOME/.tools/symbolicate-linux-fatal
 
 # swiftformat (until part of the toolchain)
 
-ARG swiftformat_version=0.40.12
+ARG swiftformat_version=0.44.0
```

+1 to Yim's suggestion

```suggestion
ARG swiftformat_version=0.44.6
```